src/components/shared/Carousel.tsx

```diff
@@ -79,19 +142,23 @@ function Carousel(props: CarouselProps): JSX.Element {
             {child &&
               <>
                 {child.topText && <h2 id={'body-text'}> {child.topText} </h2>}
-                {child.child}
-                {child.bottomText && <h2 id={'body-text'}> {child.bottomText} </h2>}
                 {child.animationTime &&
                   <span className='time-bar-container'>
                     <div key={`${reloadTime}-${slideIdx}`} className='timebar'>
                       <div className='time' style={{ '--time': child.animationTime + 's' } as CSSProperties} />
                     </div>
-                    <div onClick={() => setReloadTime(Date.now())}>
-                      <Tooltip text='Replay' >
-                        <button className='replay-button' />
-                      </Tooltip>
-                    </div>
+                    <Tooltip text= {isMuted ? 'Unmute' : 'Mute'}>
+                      <button className={'util-button ' + (isMuted ? 'unmute-button' : 'mute-button')} onClick={handleMuteButtonClick} />
+                    </Tooltip>
+                    <Tooltip text='Autoplay'>
+                      <button className={'util-button autoplay-button-' + (isAutoAdvance ? 'autoplaying' : 'not-autoplaying')} onClick={() => handleAutoplayButtonClick()} />
```

Comment:
nit:
```suggestion
                      <button className={'util-button autoplay-button-' + (isAutoAdvance ? 'autoplaying' : 'not-autoplaying')} onClick={handleAutoplayButtonClick} />
```
for replay also